fl4health/clients/basic_client.py

```diff
@@ -699,6 +700,8 @@ def validate(self) -> Tuple[float, Dict[str, Scalar]]:
                 self.test_loader, self.test_loss_meter, self.test_metric_manager, LoggingMode.TEST
             )
             # There will be no clashes due to the naming convention associated with the metric managers
+            if self.num_test_samples is not None:
+                val_metrics["test - num_examples"] = int(self.num_test_samples)
```

Comment:
Is the `int` cast here necessary given that `self.num_test_samples` has type int after the if statement?

Reply:
Was trying to use this to bypass the mypy issues, I will try both with/without as I forgot if this was necessary